### PR TITLE
upgraded Helm chart to v 1.3 and changed owner

### PR DESCRIPTION
**Pull request by MikkelHegn**
Following #62 this is the upgrade to the Helm chart.
Also updated the Helm chart owner to @mingqishao

@mingqishao / @feiskyer - what do you think about the fact that the Helm chart has the latest version, which I've overwritten? Is this a thing we should not do?

### CODE REVIEW

**Comment by MikkelHegn** [comment]
Fixes #69

**Comment by MikkelHegn** [comment]
The change in the output path for the tgz file was made to avoid permission issues when running locally. I don't see this target being run as par tof any pipeline.

**Comment by MikkelHegn** [comment]
HOLD merging until I've verified, once a new image with the 1.3.0 tag is ready.

**Comment by MikkelHegn** [comment]
Good to go!

File: Makefile [hunk 1]
```diff
@@ -68,4 +68,4 @@ bin/%:
 helm: bin/virtual-kubelet.tgz
 
 bin/virtual-kubelet.tgz:
-	rm -rf /tmp/virtual-kubelet && mkdir /tmp/virtual-kubelet && cp -r helm/* /tmp/virtual-kubelet/ &&  tar -zcvf bin/virtual-kubelet.tgz -C /tmp virtual-kubelet
+	rm -rf /tmp/virtual-kubelet && mkdir /tmp/virtual-kubelet && cp -r helm/* /tmp/irtual-kubelet/ &&  tar -zcvf virtual-kubelet.tgz -C /tmp virtual-kubelet
```
Comment (feiskyer) [inline Makefile:71]: `/tmp/irtual-kubelet/` should be `/tmp/virtual-kubelet/`

**Review by feiskyer — APPROVED** [review]

**Pull request merged**